github/event_types.go

```diff
@@ -942,6 +942,77 @@ type ProjectColumnEvent struct {
 	Installation *Installation `json:"installation,omitempty"`
 }
 
+// ProjectV2Event is triggered when there is activity relating to an organization-level project.
+// The Webhook event name is "projects_v2"
```

Comment:
nit: Please add punctuation to godoc-style comments.

```suggestion
// The Webhook event name is "projects_v2".
```